src/test/java/com/conveyal/gtfs/validator/SpeedTripValidatorTest.java

```diff
@@ -112,6 +113,12 @@ public void tripTravelingTooSlowWithMissingStopTimesHasError() {
         checkFeedHasError(TRAVEL_TOO_SLOW, "6", 3);
     }
 
+    @Test
+    public void tripHasDuplicateArrivalTimesHasError() {
+        checkFeedHasError(REPEATED_ARRIVAL_TIME,"7",2);
```

Comment:
If I were nitpicking more I'd say let's have a few more tests here to test all cases. But I'm not!